compat/src/portals.js

```diff
@@ -39,7 +39,7 @@ function Portal(props) {
 			nodeType: 1,
 			parentNode: container,
 			childNodes: [],
-			contains: () => true,
+			isPortal: true,
```

Comment:
This is an unsafe change as other faux roots could be using this, also `contains` is not a hack, it's a DOM built-in